### ISSUE TITLE
Installing this messes up IntelliSense

### ISSUE THREAD

**Issue by Theformand** [Unity-Technologies/vscode-unity-debug#7]
Hi!
Im really loving this, thank you. However upon installation of this extension (via the vscode ext install command), IntelliSense starts acting weird with code structures like if(), for() and switch().

Usually you would type: for-TAB to insert a for loop. However, now the IntelliSense list has a "for" property as the default choice. Same goes for if() and switch(). This means you have to arrow-down through the list every time you want to auto complete one of these. Screenshots of the issue here:
[1](http://puu.sh/mUyxm/5cefe2f1e7.png)
[2](http://puu.sh/mUyuD/05b882574c.png)
[3](http://puu.sh/mUyw7/44f06a280d.png)

Steps to reproduce:

1. Install VS Code (v 0.10.6 currently)
2. Install the Unity Debugger extenstion by ctrl+T and typing "ext install Unity Debugger".
3. Restart VS Code as prompted.
Try typing any of these structs.

I'm on Windows 10, Unity 5.2.2f1

**Comment by lukaszunity** [comment]
This sounds kinda odd. The extension is implemented as a separate executable that should only be used and launched while debugging, there is no integration with the code editor of any kind.

The extension is installed in %USERPROFILE%\.vscode\extensions\ and it has a bin folder where the UnityDebug.exe is located along with the assemblies (.dlls) it requires. You could try to delete some of the .dlls and see if that fixes the problem. The ICSharpCode.NRefactory / Mono.Cecil .dlls are probably the most likely ones to cause problems.

I don't think I can fix this issue in any way, as it sounds like a problem how VS Code handles the extension and not a problem with the extension it self.

**Comment by Theformand** [comment]
Hi!

I'm pretty sure I'm mistaken here. I just did a fresh install today. The messed up IntelliSense happens after installing the VSCode Unity Package from the Asset Store. Right after that, I experienced the same thing again. Funny thing is, without that package VSCode didnt recognize stuff like Mathf, or UnityEngine.Random. VSCode still seems a bit premature for me. I would love to switch out the full VS with this lightweight editor some day. Anyways, thanks for replying, I dont think this issue is with your debugger.
